turbinia/workers/__init__.py

```diff
@@ -247,6 +284,20 @@ def __init__(
     self.user = user if user else getpass.getuser()
     self._evidence_config = {}
 
+  def serialize(self):
+    """Converts the TurbiniaTask object into a serializable dict.
+
+    Returns:
+      Dict: Dictionary representing this object, ready to be serialized.
+    """
+    orig = (self.output_manager, self.last_update)
+    self.output_manager = self.output_manager.__dict__
```

Comment:
I know this currently only gets called right before its returned, but maybe we should do the copy at the top of this method, and then we can just update the `output_manager` and `last_update` attributes on the copy rather than having to make a separate copy of them.  This would also mean that we won't have as drastic side-effects of calling this method (in case we want to call this method elsewhere later) since we won't have to overwrite part of the current object.

Reply:
Yep, that definitely simplifies this.